m4_sessions/1706-inning/group_four.md

```diff
@@ -1,63 +1,77 @@
-# Group Four
+07/14/17
 
-### Session Title:
+## Gear-up title: Being a Newb in Tech (aka Don't be a Selfish Bastard)
 
-"Getting involved and giving back as a beginner"  
+### Tagline:
 
-### Session Outcomes: 
+_Do you think you're too new to make contributions to those around you? You're not, you can start tomorrow, and everyone (yourself included) will be better for it!_
 
-1. Doc to add to their portfolio
-2. Interest/list of OSS to make first PR
-3. Check list to work through for first PR (fulfills PD deliverables, including [blog post (all mods), PR (mod 4)])
+#### Other possible titles:
+- Giving back, even (especially) when you think you can't
+- Helping others for fun and profit
+- How not to be a selfish bastard
+- leverage your inner newb for good
+- making contributions in an industry where that's rewarded
 
-Feelings and thoughts:
 
-- people feel confident in their ability to contribute to the community
-- people have the knowledge of where to go to contribute
-- the knowledge of HOW to contribute (like OSS PR)
 
+# Session outline:
```

Comment:
Might want to just intro the team and also be sure to give them time to do some introductions!